112-running-multiple-versions-of-connector-plugins.md

```diff
@@ -0,0 +1,123 @@
+# Running multiple versions of Connector plugins
+
+In Apache Kafka 4.1.0 onwards it's possible to run multiple versions of the same Connector plugin on the same Connect cluster.
+Support for this was added via [KIP-891][kip_891].
+
+Strimzi should support this feature as part of the `KafkaConnector` CR.
+
+## Current situation
+
+Today Strimzi provides the `KafkaConnector` CR which looks something like:
+
+```yaml
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaConnector
+metadata:
+  name: my-source-connector
+  labels:
+    strimzi.io/cluster: my-connect-cluster
+spec:
+  class: org.apache.kafka.connect.file.FileStreamSourceConnector
+  tasksMax: 2
+  config:
+    file: "/opt/kafka/LICENSE"
+    topic: my-topic
+```
+
+If there are multiple versions of the same connector loaded into the Connect cluster then Connect chooses the version to use.
+This is based on the [Maven Version Order](https://maven.apache.org/pom.html#Version_Order_Specification) and the latest version based on that is used.
+There are a few other rules which are listed in [KIP-891][kip_891].
+
+## Motivation
+
+Once Strimzi supports Apache Kafka version 4.1.0, without any changes users will be able to select the version of the connector they want by specifying `connector.plugin.version` in the `config` section of the `KafkaConnector` CR.
```

Comment:
So, we already do support 4.1.0 -> it will be part of 0.48.

Reply:
We do in main but not in a released version of Strimzi, so I just wanted to be clear for anyone who was testing